src/ansys/openapi/common/_session.py

```diff
@@ -86,7 +86,14 @@ class AuthenticationScheme(Enum):
 
     Used to specify an authentication scheme used when connecting to the server with credentials.
 
-    .. versionadded:: 2.1
+    .. only:: OpenapiCommonStandaloneBuild
+
+        .. versionadded:: 2.1
+
+    .. only:: not OpenapiCommonStandaloneBuild
+
+        .. tip::
+           Added to :doc:`ansys-openapi-common <openapi-common:index>` in version 2.1.
```

Comment:
We could change this to just ``ansys-openapi-common`` and remove the link, but I feel like the additional burden of having to use the same intersphinx reference is a small price to pay for improved crosslinking between docs

Reply:
Irrelevant now we link to the api docs directly.